masterlist.yaml

```
@@ -3269,6 +3269,33 @@ plugins:
       - crc: 0xCF743DF7
         util: 'TES4Edit v4.0.3h (Hotfix 1)'
 
+  - name: 'StarX Vampire Deaths\.es(m|p)'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/25613/'
+        name: 'StarXs Vampire Deaths'
+      - link: 'https://www.nexusmods.com/oblivion/mods/50480/'
+        name: 'StarX Vampire death Improved Extended Edition'
+    msg:
+      - <<: *patchUpdateAvailable
+        subs: [ '[StarX Vampire death Improved Extended Edition](https://www.nexusmods.com/oblivion/mods/50480/)' ]
+        condition: 'file("StarX Vampire Deaths.esm")'
+  - name: 'StarX Vampire Deaths.esm'
```

Considering the condition, it'd be better to just apply the message to the ESM on its own.

```suggestion
  - name: 'StarX Vampire Deaths.esm'
    msg:
      - <<: *patchUpdateAvailable
        subs: [ '[StarX Vampire death Improved Extended Edition](https://www.nexusmods.com/oblivion/mods/50480/)' ]
```